camunda-formio-tasklist-vue/src/components/layout/LeftSider.vue

```diff
@@ -165,6 +167,10 @@ export default class LeftSider extends Mixins(BaseMixin) {
     return getFormattedDateAndTime(date);
   }
 
+  getTimeAndTime(date: Date){
```

Comment:
`getTimeAndTime` doesn't look approapriate for this function.

Can you rename it is as `getDateAndTime` or something better?

Reply:
changed getLocalStringDate